UIKitDZ/UIKitDZ/StoryBoardViewController.swift

```diff
@@ -0,0 +1,12 @@
+// StoryBoardViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// StoryBoardViewController
+class StoryBoardViewController: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        print("ds")
+    }
+}
```

Comment:
```suggestion
final class StoryBoardViewController: UIViewController {}
```
Можно просто убрать метод.